doc/modules/compose.rst

```diff
@@ -117,16 +117,19 @@ permitted). This is convenient for performing only some of the transformations
 
 .. _pipeline_nested_parameters:
 
-|details-start|
-**Nested parameters**
-|details-split|
+Nested parameters
```

Comment:
The section is actually looking weird. I don't think we should have collapsed any of the information.

Right now, we don't have an example in the documentation that show how to make a pipeline.